pkgs/development/python-modules/deprecation/default.nix

```diff
@@ -1,10 +1,8 @@
-{ lib, buildPythonPackage, fetchPypi
+{ lib
+, buildPythonPackage
+, fetchPypi
 , fetchpatch
 , packaging
-, python
-, pythonAtLeast
-, pythonOlder
-, unittest2
 , unittestCheckHook
 }:
 
```

Comment:
We should set disabled for pythons older than 3.10.

Reply:
It still works in those older versions, because https://github.com/briancurtin/deprecation/pull/57 that we pull in conditionally imports `unittest2`.